netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -390,6 +390,42 @@ _ebpf_sock_addr_set_redirect_context(_In_ const bpf_sock_addr_t* ctx, _In_ void*
     return return_value;
 }
 
+static uint64_t
+_ebpf_sock_addr_get_current_process_start_key(
```

Comment:
feedback from ebpf meeting: these should be moved to a global function in ebpf-core